example/src/domain/Expand.js

```diff
@@ -0,0 +1,15 @@
+import { useElementSize } from '@kaliber/use-element-size'
```

Comment:
Zullen we deze niet in de index zetten? Dan is het makkelijker te begrijpen

Reply:
Yes, goed idee inderdaad